src/test/java/project/web/GasStationControllerTest.java

```diff
@@ -74,11 +74,10 @@ void getGasPriceInfoTestWithInvalidInput() {
         String fuelType = "Diesel";
         when(gasStationService.getGasPriceInfo(fuelType)).thenReturn(null);
 
-        // WHEN
-        ResponseEntity<GasPriceInfoDTO> response = gasStationController.getGasPriceInfo(fuelType);
-
-        // THEN
-        assertEquals(HttpStatus.NOT_FOUND, response.getStatusCode());
+        // WHEN THEN
+        assertThat(gasStationController.getGasPriceInfo(fuelType))
```

Comment:
same here, please do not do it if possible

Reply:
fixed